tahrir/views.py

```diff
@@ -70,6 +70,47 @@ def _get_user(request, id_or_nickname):
             return None
 
 
+def _get_user_badge_info(request, user):
+    """ Returns a dictionary of the user badge information
+    """
+
+    # Get user badges.
+    user_badges = [a.badge for a in user.assertions]
+
+    # Sort user badges by id.
+    user_badges = sorted(user_badges, key=lambda badge: badge.id)
```

Comment:
FWIW, if you really wanted to avoid the lambda you could use:

```python
import operator
user_badges = sorted(user_badges, key=operator.attrgetter('id'))
```

The lambda definitely works!